Dockerfile

```diff
@@ -1,35 +1,70 @@
-FROM python:3.9.7-slim@sha256:cd1045dbabff11dab74379e25f7974aa7638bc5ad46755d67d0f1f1783aee101
+# syntax=docker/dockerfile:1.3
 
-ENV PYTHONUNBUFFERED=1 \
-    PYTHONPATH=/app/ \
-    PORT=8888
+FROM python:3.9-slim-bullseye as compile
```

Comment:
What do you think about specifying the hash of the version of the image here too? Would potentially save us from some random breaking changes that are made to the base image.